spec/features/stops_controller_spec.rb

```diff
@@ -13,8 +13,15 @@
       new_stop = Stop.create(name: "Porter")
       get :show, params: {id: new_stop.id}
       parsed = JSON.parse(response.body)
-      expect(parsed["name"]).to eq("Porter")
+      expect(parsed["stop"]["name"]).to have_content("Porter")
     end
-  end
 
+    it "returns an associated review" do
+      new_stop = Stop.create(name: "Porter")
+      new_review = Review.create(rating: 4, body: "this is a wendys", stop_id: new_stop.id)
+      get :show, params: {id: new_stop.id}
+      parsed = JSON.parse(response.body)
+      expect(parsed["reviews"][0]["body"]).to have_content("this is a wendys")
```

Comment:
I love this review. It made me laugh.

Reply:
We will now change it to something that doesn't make you laugh